tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
```

Comment:
That's an unexpected side-effect of importing "dbus.sh". I can imagine people being somewhat frustrated looking for it. Why do we need it here?